### PR TITLE
Wait for the results (HTTPObservatory)

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: modules/checks/httpobservatory.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ module.exports = async () => {
     const name = 'HTTP Observatory'
     async function tryBlock(page) {
       await page._client.send('Emulation.clearDeviceMetricsOverride')
-      await page.goto('https://observatory.mozilla.org/analyze/' + url + '&third-party=false', { waitUntil: 'networkidle0' })
-      await page.waitForFunction('!document.querySelector("#scan-progress-bar")', { timeout: 240000 })
+      await page.goto('https://observatory.mozilla.org/analyze/' + url + '?third-party=false', { waitUntil: 'networkidle0' })
```
Comment (kkrieger85) [inline modules/checks/httpobservatory.js]: What if someone wants to check a specific (wordpress) page without cool speaking urls?
Like domain.tls/index.php?id=42

Comment (DanielRuf) [reply modules/checks/httpobservatory.js]: The checks are generally meant for domain names, not full URLs with query parameters.

Also see the website of HTTPObservatory and try `example.com?test=123` which will then use `example.com` so they already strip that. Same for `example.com/test123`.

The checks are only relevant for the domain (TLS certificate, server setting and so on).

Comment (DanielRuf) [reply modules/checks/httpobservatory.js]: https://observatory.mozilla.org/

Comment (DanielRuf) [reply modules/checks/httpobservatory.js]: The checks are generally meant for domain names, not full URLs with query parameters.

Also see the website of HTTPObservatory and try `example.com?test=123` which will then use `example.com` so they already strip that. Same for `example.com/test123`.

https://observatory.mozilla.org/

The checks are only relevant for the domain (TLS certificate, server setting and so on).

**Pull request merged**